init-tools.sh

```
@@ -22,18 +22,14 @@ fi
 
 if [ -e $__TOOLRUNTIME_DIR ]; then rm -rf -- $__TOOLRUNTIME_DIR; fi
 
-if [ -d "$DotNetBuildToolsDir" ]; then
+if [ "$DotNetBuildToolsDir" != "" ]; then
```

Prefer to fail rather than silently ignore when `$DotNetBuildToolsDir` is set, but not a directory.